src/chocolatey/infrastructure.app/commands/ChocolateyLicenseCommand.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright © 2017 - 2021 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey.infrastructure.app.commands
+{
+    using chocolatey.infrastructure.app.attributes;
+    using chocolatey.infrastructure.app.configuration;
+    using chocolatey.infrastructure.commandline;
+    using chocolatey.infrastructure.commands;
+    using chocolatey.infrastructure.filesystem;
+    using chocolatey.infrastructure.licensing;
+    using chocolatey.infrastructure.logging;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+
+    [CommandFor("license", "Retrieve or modify the Chocolatey License")]
+    public class ChocolateyLicenseCommand : ICommand
+    {
+        public void configure_argument_parser(OptionSet optionSet, ChocolateyConfiguration configuration)
+        {
+            // We don't currently expect to have any arguments
+        }
+
+        public void handle_additional_argument_parsing(IList<string> unparsedArguments, ChocolateyConfiguration configuration)
+        {
+            // We don't currently expect to have any additional arguments
+        }
+
+        public void handle_validation(ChocolateyConfiguration configuration)
+        {
+            // We don't currently accept any arguments, so there is no validation
+        }
+
+        public void help_message(ChocolateyConfiguration configuration)
+        {
+            this.Log().Info(ChocolateyLoggers.Important, "License Command");
+            this.Log().Info(@"
+Chocolatey will do license things.
+");
+        }
+
+        public bool may_require_admin_access()
+        {
+            return false;
+        }
+
+        public void noop(ChocolateyConfiguration configuration)
+        {
+        }
+
+        public void run(ChocolateyConfiguration config)
+        {
+            var ourLicense = LicenseValidation.validate();
+            var nodeCount = parse_node_count(ourLicense.Name);
+            var logger = config.RegularOutput ? ChocolateyLoggers.Normal : ChocolateyLoggers.LogFileOnly;
+
+            if (ourLicense.LicenseType == ChocolateyLicenseType.Foss)
```

Comment:
It looks like we can use `is_licensed_version()`: https://github.com/chocolatey/choco/blob/develop/src/chocolatey/infrastructure/licensing/ChocolateyLicense.cs#L36